vignettes/subgroup.Rmd

```diff
@@ -84,7 +84,7 @@ When you create the special classed dataset for `brms.mmrm` using `brm_data()`,
 
 ```r
 data <- brm_data(
```

Comment:
> It may be helpful for the user to be able to see in the documentation of brm_formula() what the default formula setting is:

Updated the `?brm_formula` help page in 3bffbada931938d698c6627b79fe44b91fe7bf44 to explain the defaults.

> (If the model considers a subgroup, it could be perceived as a little bit confusing to see group:subgroup. arm:subgroup or treatment:subgroup may be more intuitive. But this may be a separate discussion and not critical for this PR.)

The names of the terms in the formula are aligned with the column names in the data. In the general case, the user can choose different ones. `brms.mmrm` uses the term "group" throughout to denote treatment group, which I thought at the time was both generic and clear enough. Changing it at this point might cause inconsistencies and deprecations at many levels of the package. (Not impossible, but definitely a lot of work.)